sopel/coretasks.py

```diff
@@ -162,28 +221,41 @@ def execute_perform(bot, trigger):
 def startup(bot, trigger):
     """Do tasks related to connecting to the network.
 
-    001 RPL_WELCOME is from RFC2812 and is the first message that is sent after
-    the connection has been registered on the network.
+    ``001 RPL_WELCOME`` is from RFC2812 and is the first message that is sent
```

Comment:
I smell a future project to convert RFC references throughout the docs to Sphinx extlinks. Like:

```rst
:rfc:`2812`
```

Reply:
Perfect for yet another PR.